x-pack/plugins/dashboard_enhanced/public/services/drilldowns/dashboard_to_dashboard_drilldown/drilldown.tsx

```diff
@@ -48,7 +58,48 @@ export class DashboardToDashboardDrilldown
     return true;
   };
 
-  public readonly execute = () => {
-    alert('Go to another dashboard!');
+  public readonly execute = async (
+    config: Config,
+    context: ActionContext<VisualizeEmbeddableContract>
+  ) => {
+    console.log('drilldown execute'); // eslint-disable-line
+    const getUrlGenerator = await this.params.getGetUrlGenerator();
+    const navigateToApp = await this.params.getNavigateToApp();
+    const { timeRange, query, filters } = context.embeddable.getInput();
+
+    // @ts-ignore
+    if (context.data.range) {
+      // look up by range
+      const { restOfFilters, timeRangeFilter } =
+        (await selectRangeActionGetFilters({
+          timeFieldName: context.timeFieldName,
+          data: context.data,
+        })) || {};
+      console.log('select range action filters', restOfFilters, timeRangeFilter); // eslint-disable-line
+      // selectRangeActionGetFilters
+    } else {
+      const { restOfFilters, timeRangeFilter } = await valueClickActionGetFilters({
+        timeFieldName: context.timeFieldName,
+        data: context.data,
+      });
+      console.log('value click action filters', restOfFilters, timeRangeFilter); // eslint-disable-line
+    }
+
+    const dashboardPath = await getUrlGenerator(DASHBOARD_APP_URL_GENERATOR).createUrl({
+      dashboardId: config.dashboardId,
+      query,
+      // todo - how to get destination dashboard timerange?
```

Comment:
Do we need it? 
I think how it is works right now: 

1. if to pass time in the url - it will be applied
2. If to not pass time in the url: 
2.1 If destination dashboard has time range saved with it -> then it will be applied
2.2 if destination dashboard doesn't have any time range saved with it -> default will be applied (e.g. last 15 min)